tests/cephfs/mds_backtrace_damage.py

```diff
@@ -0,0 +1,154 @@
+        fs_util.kernel_mount(
+            [client1],
+            kernel_mounting_dir_1,
+            ",".join(mon_node_ips),
+            extra_params=f",fs={fs_name}",
+        )
+        fs_util.fuse_mount(
+            [client1], fuse_mounting_dir_1, extra_params=f" --client-fs={fs_name}"
+        )
+        dir_name = "".join(
+            random.choice(string.ascii_lowercase + string.digits)
+            for _ in list(range(5))
+        )
+        mount_test_case(clients, fuse_mounting_dir_1)
+        mount_test_case(clients, kernel_mounting_dir_1)
+        for i in range(num_dir):
+            client1.exec_command(
+                sudo=True, cmd=f"mkdir -p {fuse_mounting_dir_1}/{dir_name}_{i}"
+            )
+            client1.exec_command(
+                sudo=True, cmd=f"mkdir -p {kernel_mounting_dir_1}/{dir_name}_{i}"
+            )
+            for k in range(num_files):
+                client1.exec_command(
+                    sudo=True,
+                    cmd=f"touch {fuse_mounting_dir_1}/{dir_name}_{i}/file_{k}",
+                )
+                client1.exec_command(
+                    sudo=True,
+                    cmd=f"touch {kernel_mounting_dir_1}/{dir_name}_{i}/file_{k}",
+                )
+        time.sleep(30)
+        fs_util.run_ios(client1, fuse_mounting_dir_1, ["dd", "smallfile"])
+        fs_util.run_ios(client1, kernel_mounting_dir_1, ["dd", "smallfile"])
+        time.sleep(30)
+        out_fs, _ = client1.exec_command(sudo=True, cmd="ceph fs status")
+        print(out_fs)
+        # remove the "parent" attribute from the objects
+        object_list, _ = client1.exec_command(
+            sudo=True, cmd=rf'rados -p {pool_name_meta} ls | grep  "\.00000000"'
+        )
+        log.info(object_list)
+        object_list = object_list.split("\n")
+        number_of_objects = len(object_list)
+        log.info("number of objects: " + str(number_of_objects))
+        for obj in object_list:
+            if str(obj) != "":
+                log.info("obj name:" + str(obj))
+                xttr, _ = client1.exec_command(
+                    sudo=True, cmd=f"rados -p {pool_name_meta} listxattr {str(obj)}"
+                )
+                log.info(xttr)
+                if "parent" in xttr:
+                    client1.exec_command(
+                        sudo=True, cmd=f"rados -p {pool_name_meta} rmxattr {obj} parent"
+                    )
+        # run scrub
+        client1.exec_command(sudo=True, cmd=f"ceph tell mds.{fs_name}:0 scrub start /")
+        # check if the cluster is in damaged state
+        time.sleep(30)
+        out1, _ = client1.exec_command(sudo=True, cmd="ceph -s")
+        print(out1)
+        out2, _ = client1.exec_command(sudo=True, cmd="ceph fs status")
+        print(out2)
+        if "HEALTH_ERR" in out1:
+            log.info("Cluster is in damaged state")
+            return 0
+        else:
+            log.info("Cluster is not in damaged state")
```

Comment:
The else part should be reported as log.error if the cluster does not reach damaged state?

Reply:
I have changed it to `log.error()`
thanks